cmd/address_gen/address_gen.go

```diff
@@ -46,11 +65,29 @@ func main() {
 		os.Exit(1)
 	}
 
-	output, err := json.MarshalIndent(w, "", "    ")
+	if *verbose {
+		output, err := json.MarshalIndent(w, "", "    ")
+		if err != nil {
+			fmt.Println("Error formating wallet to JSON. Error:", err)
+			os.Exit(1)
+		}
+
+		fmt.Println(string(output))
+		return
+	}
+
+	// only show the address, no more info
+	addrs := make([]string, 0, len(w.Entries))
+
+	for _, e := range w.Entries {
+		addrs = append(addrs, e.Address)
+	}
+
+	v, err := json.MarshalIndent(addrs, "", "    ")
```

Comment:
Why marshal array as json? Isn't printing `for _, a := range addrs { fmt.Println(a) }` ok?